magefiles/magefile.go

```diff
@@ -855,6 +856,38 @@ func CleanWebHooks() error {
 	return nil
 }
 
+// Remove all webhooks which with 1 day lifetime from Gitlab rpo.
+func GitlabCleanWebHooks() error {
+	gcToken := utils.GetEnv(constants.GITLAB_TOKEN_ENV, "")
+	if gcToken == "" {
+		return fmt.Errorf("empty PAC_GITLAB_TOKEN env")
+	}
+	projectID := utils.GetEnv(constants.GITLAB_PROJECT_ID, "")
+	if projectID == "" {
+		return fmt.Errorf("empty PAC_PROJECT_ID env. Please provide a valid GitLab Project ID")
+	}
+	gitlabURL := utils.GetEnv(constants.GITLAB_URL_ENV, "https://gitlab.com/api/v4")
+	gc, err := gitlab.NewGitlabClient(gcToken, gitlabURL)
+	if err != nil {
+		return err
+	}
+	webhooks, _, err := gc.GetClient().Projects.ListProjectHooks(projectID, nil)
+	if err != nil {
+		return fmt.Errorf("failed to list project hooks: %v", err)
+	}
+	// Delete webhooks were created older than 1 day
```

Comment:
Delete webhooks were created older than 1 day
```suggestion
	// Delete webhooks that are older than 1 day
```

Reply:
Updated.